lib/sms/sms_cms_codes.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _SMS_ERROR_STR_INCLUDE_H_
+#define _SMS_ERROR_STR_INCLUDE_H_
+
+/**
+ * @brief Retrieve a statically allocated textual description for a given CMS error reason.
+ *
+ * @param reason CMS error reason.
+ * @return const char* CMS error reason description.
```

Comment:
I don't think we need a type in the return value comments.

Reply:
removed